include/Track.h

```diff
@@ -209,6 +209,12 @@ public slots:
 
 	void toggleSolo();
 
+protected:
+	bool isPresetMode() const
```

Comment:
This could go in one line to save space (but it is OK like that, too).

Reply:
Adjusted with commit https://github.com/LMMS/lmms/pull/7445/commits/a1c276debf59d278182f0ebd9658e43c70bca015.